corehq/apps/case_importer/do_import.py

```diff
@@ -250,6 +259,17 @@ def _populate_updated_fields(self, row):
             elif update_value is not None:
                 update_value = _convert_field_value(update_value)
 
+            if update_field_name in self.validated_fields:
+                case_property = self.validated_fields[update_field_name]
+                if not case_property.valid_value(update_value):
+                    # note: probably want to accumulate full list of invalid
```

Comment:
Yes, agreed.

Reply:
![image](https://user-images.githubusercontent.com/430048/125868475-663b87cf-ef49-4333-9f8b-08c5b5537191.png)

Now more than one error will be reported on a row (e.g. row 3 here had multiple columns with date errors).